src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,

```

Comment:
```suggestion
public enum Color {
    RED,
```

![image](https://github.com/mate-academy/jv-lottery/assets/49767049/f3b903ac-d4c7-404e-aae3-c3eaf32c5076)